nilearn/maskers/tests/test_multi_nifti_masker.py

```diff
@@ -184,13 +184,11 @@ def test_mask_strategy_errors(list_random_imgs):
 )
 def test_compute_mask_strategy(strategy, shape_3d_default, list_random_imgs):
     """Check different strategies to compute masks."""
-    masker = MultiNiftiMasker(mask_strategy=strategy, mask_args={"opening": 1})
+    masker = MultiNiftiMasker(mask_strategy=strategy)
```

Comment:
I was wondering why you changed that?

Reply:
I think I made this change. Basically, setting `force_resample` to True in the resample function has changed the expected "dummy" mask outputs.

We expected all zeros in the resulting mask, but having this parameter resulted in some 1s only with "wm" masking strategy.

To me, the test is more of a smoke test, and I don't see the reason to have the opening argument there. (The input image is just a bunch of random numbers, so I am not sure how one could predict the output mask)